src/reducers/MapReducer.js

```diff
@@ -211,6 +212,9 @@ export default (state = INITIAL_STATE, action) => {
 
             return {...state, selectedSite};
 
+        case SELECTED_SITE_CLEARED:
+            return {...state, selectedSite: INITIAL_STATE.selectedSite}
```

Comment:
When we pass a `selectedSite: null`, the action fires off this `SELECTED_SITE_CLEARED` which tells us to clear out the `selectedSite` to the `INITIAL_STATE` version, which is just an empty object.